scipy/optimize/_linprog_highs.py

```diff
@@ -338,24 +426,23 @@ def _linprog_highs(lp, solver, time_limit=None, presolve=True,
     A = csc_matrix(A)
 
     options = {
-        'presolve': presolve,
-        'sense': HIGHS_OBJECTIVE_SENSE_MINIMIZE,
-        'solver': solver,
```

Comment:
I'd prefer to keep the `black`-style formatting changes out of this PR, as it makes it much harder to follow what changed in a meaningful way, and what just got some formatting changes.

Reply:
Taking another look at the `.pyx` file there's really almost nothing which gets brought forward, so I'm not sure what the goal would be, the file is both much smaller and logically restructured as well... Is there a particular part of the `pyx` file which seems to be relevant? This would basically require reworking the PR from scratch for what would seem to be very little gain, a full rewrite to `highspy` is IMO the right time to switch formatting as well...

Reply:
> This would basically require reworking the PR from scratch...

By which I mean restoring the `pyx` file, renaming it, and then deleting / rewriting in a series of new commits.

Reply:
I tried this actually, and it will still show up as deleted in the final (consolidated) `git` diff, not moved (even though the earlier commits now [show a rename](https://github.com/scipy/scipy/pull/19255/commits/b19e74d5a723d07c114309d949f8137f1fcfee1b)).